src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,21 @@
 // your components will all go in this `component` directory.
 // feel free to change this component.js into TodoList.js
+import React from 'react';
+import Todo from './Todo';
+
+
+
+const TodoList = props => {
+    
+    return (
+        <div className="Todo-list">
+            {props.todos.map((todoX) => {
+                return (
+                  <Todo todo={todoX} key={todo.id} />
+                )
```

Comment:
I got a ReferenceError when I tried to run the code because `todo is not defined` on `TodoList.js`. It is because of this component:

`<Todo todo={todoX} key={todo.id} />`, `todo.id` should be `todoX.id`